### PR TITLE
Add menu feature on mobile verision

### PR DESCRIPTION
**Pull request by gergitokusse**
### The following file was added 
👉 Linter configuration for HTML+CSS+JS
👉 new Jscript.js file for the menu feature
👉 JS DOM was applied

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,15 +32,16 @@
...
             alt="hamburguer icon"
           />
-          <ul class="navigation_bar menu-list">
-            <li><a href="#works" >Portfolio</a> </li>
-            <li><a href="#about" >About</a></li>
-            <li><a href="#contact-me" >Contact</a></li>
+          <ul class="menu-list non-visible">
+            <li><a class="li-a" href="#works" >Portfolio</a> </li>
+            <li><a class="li-a2" href="#about" >About</a></li>
+            <li><a class="li-a3" href="#contact-me" >Contact</a></li>
```
Comment (tugsem) [inline index.html:44]: - [ ] Please ensure in the mobile menu, overflow is hidden, it covers the whole page and has a blurry background as in the given figma design:
- figma
image
- your design
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @gergitokusse  and @techmoves 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️linters are passing
✔️correct use of github flow
✔️professional documentation
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (gergitokusse) [reply index.html:44]: ###  👍 👍 👍  Okay thank you much for your nice and constructive comment. 👍 👍 👍

File: style.css [hunk 2]
```diff
@@ -204,6 +204,64 @@ body {
...
+  justify-content: start;
+  background-color: #0113d6;
+  color: white;
+  height: 812px;
+  padding: 5px;
+  opacity: 65%;
+  transition: all;
+  transition-duration: 0.75s;
+  backface-visibility: hidden;
+}
```
Comment (thecodechaser) [inline style.css]: - [ ] As requested by the previous reviewer, The mobile menu should cover the whole height, Currently at the bottom I can see the page content. Please make sure to fix this issue.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @gergitokusse, and Team👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (gergitokusse) [reply style.css]: ### Okay I am so sorry I forgot it, wait a moment  I will fix it in a short time.

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @gergitokusse, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewers. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




### To Highlight :1st_place_medal:

- Descriptive Pull Request title and summary. ✔️
- Has Hamburger Menu functionality ✔️
- Mobile menu options, the correct part of the page is displayed ✔️
- Good use of Github flow ✔️
- Implemented reviewer changes requested ✔️ 


## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**